cypress/README.md

```diff
@@ -0,0 +1,23 @@
+## RHOSE UI - Integration tests
+
+This project contains Cypress test suites to verify UI of RHOSE.
+
+All steps should be performed from the root folder (it means `cd ..`).
```

Comment:
minor thing, maybe too personal opinion, I would just instruct: `... from the sandbox-ui root folder`